lib/unsupported-features/node-builtins-modules/repl.js

```diff
@@ -0,0 +1,49 @@
+"use strict"
+
+const { CALL, READ } = require("@eslint-community/eslint-utils")
+
+/** @type {import('../types.js').SupportVersionTraceMap} */
+const repl = {
+    start: {
+        [READ]: { supported: ["0.1.91"] },
+    },
+    writer: {
+        [READ]: { supported: ["0.1.91"] },
+    },
+    REPLServer: {
+        [READ]: { supported: ["0.1.91"] },
+        [CALL]: { deprecated: ["22.9.0"] },
+    },
+    REPL_MODE_MAGIC: {
+        [READ]: {
+            supported: ["4.0.0"],
+            deprecated: ["8.0.0"],
+            // removed: ['10.0.0'],
```

Comment:
I am not sure if we want to include this data point, but I left it here just in case.